src/routes/routes.ts

```diff
@@ -123,3 +124,10 @@ export const generatePrefixedAddressRoutes = (params: SafeRouteParams): typeof S
     {} as typeof STANDARD_SAFE_ROUTES,
   )
 }
+
+const DEMO_SAFE_MAINNET = '0xfF501B324DC6d78dC9F983f140B9211c3EdB4dc7'
+
+export const demoSafeRoute = generateSafeRoute(SAFE_ROUTES.APPS, {
+  shortName: 'eth',
+  safeAddress: DEMO_SAFE_MAINNET,
+})
```

Comment:
I would move the `DEMO_SAFE_MAINNET` to the `src/utils/constants` file. The `demoSafeRoute` can stay where it is used. Sorry, perhaps I wasn't clear before.

Reply:
For some reasons when I moved the `DEMO_SAFE_MAINNET` to the `src/utils/constants` file, some units tests in the non-related `googleTagManager.test.tsx` file started to fail:

After some research it seems to be related with the way that this mock its defined:

```
  describe('loadGoogleTagManager', () => {
    it('prevents init without a gtm id/auth', () => {
      jest.doMock('src/utils/constants.ts', () => ({
        GOOGLE_TAG_MANAGER_ID: '',
        GOOGLE_TAG_MANAGER_DEVELOPMENT_AUTH: '',
      }))

     [....]
```

I updated them to only mock the values that are required for that tests using `jest.requireActual()`:

```
      jest.doMock('src/utils/constants.ts', () => {
        const original = jest.requireActual('src/utils/constants.ts')
        return {
          ...original,
          GOOGLE_TAG_MANAGER_ID: 'id123',
          GOOGLE_TAG_MANAGER_DEVELOPMENT_AUTH: 'auth123',
        }
      })
```

Now it seems to be working but I think that someone should have a further look into it.